whois-api/src/main/java/net/ripe/db/whois/api/nrtm4/WhoisNrtmServletDeployer.java

```diff
@@ -25,9 +25,15 @@ public WhoisNrtmServletDeployer(final NrtmClientService nrtmClientService, final
 
     @Override
     public void deploy(final WebAppContext context) {
+        final JacksonJsonProvider jacksonJsonProvider = new JacksonJsonProvider();
+        jacksonJsonProvider.configure(SerializationFeature.WRITE_EMPTY_JSON_ARRAYS, true);
+        jacksonJsonProvider.configure(SerializationFeature.INDENT_OUTPUT, true);
```

Comment:
What's the effect of this change? Does all JSON get reformatted?

Reply:
Sorry again, I was testing something I again added these lines by mistake. I will remove them